docs/api.md

```diff
@@ -25,6 +25,7 @@ Content-Type: application/json
 | `vCodec`          | `string`  | `h264 / av1 / vp9`                 | `h264`    | applies only to youtube downloads. `h264` is recommended for phones.            |
 | `vQuality`        | `string`  | `144 / ... / 2160 / max`           | `720`     | `720` quality is recommended for phones.                                        |
 | `aFormat`         | `string`  | `best / mp3 / ogg / wav / opus`    | `mp3`     |                                                                                 |
+| `aQuality`         | `string`  | `320 / 192 / 128 / 64`    | `320`     | set audio bitrate|
```

Comment:
isnt that an int?

Reply:
actually, its a string and I did that intentionally as every other similar param was a string too and there isn't a need for this to be int tbh

Reply:
the description for this is not very explanatory, maybe something like: "adjusts the bitrate of the converted output file when downloading audio"

Reply:
updated it as required